modular_zubbers/code/modules/branding/brand.dm

```diff
@@ -0,0 +1,73 @@
+/obj/item/knife/brand
+/* 	icon = 'icons/obj/stack_objects.dmi'
+	icon_state = "rods-1" */
+	name = "branding knife"
+	icon = 'modular_skyrat/modules/modular_ert/icons/pizza/hotknife.dmi'
+	icon_state = "hotknife"
+	inhand_icon_state = "hotknife"
+	desc = "Once known as Lightbringer, this sword has been demoted to a simple pizza cutting knife... It may still have its fire attack powers."
+	righthand_file = 'modular_skyrat/modules/modular_ert/icons/pizza/righthand.dmi'
+	lefthand_file = 'modular_skyrat/modules/modular_ert/icons/pizza/lefthand.dmi'
+//	var/list/static/possible_crimes = list()
+
+/obj/item/knife/brand/attack(mob/living/victim, mob/living/attacker, params)
+	var/list/branding_with = typesof(/datum/status_effect/branded)
+	branding_with -= /datum/status_effect/branded
+/* 	var/list/preset_brandable_crimes = list()
+	for(var/i in branding_with)
+		preset_brandable_crimes |= i.initial(id) */
+	if(!istype(victim, /mob/living/carbon))
+		return
+	var/datum/status_effect/branded/current_branding = tgui_input_list(attacker, "What crime are you branding them with?", "Branding", branding_with) // todo, make names show
+
+	if(!attacker.combat_mode && do_after(attacker, 6 SECONDS, victim))
+		victim.apply_status_effect(current_branding)
+		to_chat(attacker, span_userdanger("You brand [victim] with the mark of [initial(current_branding.id)]"))
+
+		var/obj/item/bodypart/branded_limb = victim.get_bodypart(deprecise_zone(BODY_ZONE_CHEST))
+		branded_limb.force_wound_upwards(/datum/wound/burn/flesh/critical, wound_source = "Branding")
```

Comment:
```
victim.cause_wound_of_type_and_severity(WOUND_BURN, branded_limb, WOUND_SEVERITY_CRITICAL))
```

same effect except it supports synthetic wounds

Reply:
you should use this anyway, this is the defacto "give a limb a wound" proc

source: i refactored wounds and added t his proc